src/App.js

```diff
@@ -1,19 +1,60 @@
 import "./App.scss";
+import React, { useState } from "react";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
-import Icon from './components/Icon/Icon';
+import WeatherIcon from "./components/Picture/WeatherIcon";
+import CurrentWeather from "./components/CurrentWeather/CurrentWeather";
+import FutureWeather from "./components/FutureWeather";
+
 
-//configs
 const siteTitle = process.env.REACT_APP_SITE_TITLE ?? "CYF Weather";
 
 function App() {
+  const [weatherData, setWeatherData] = useState([]);
+  
+  function getNewWeather(city) {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/forecast?q=${city}&units=metric&appid=3b86046cce0de3be7cfa8369f4540b37`
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        setWeatherData(data);
+      });
+  }
+  
   return (
     <div className="App">
-      <Header title={siteTitle} />
+      <Header
+        title={siteTitle}
+        getNewWeather={getNewWeather}
+        weatherData={weatherData}
+      />
+
       <main className="c-site-main" tabIndex="0">
-      <Icon name="clear"/>
-    
+        <section>
+          <WeatherIcon weatherId={weatherData?.list?.[0]?.weather?.[0]?.id} />
```

Comment:
Why is this WeatherIcon component not part of CurrentWeather? It displays information about the current weather so it should appear in the same component that displays the temp, pressure etc.

Move it down one level inside the CurrentWeather component

Reply:
I moved it down inside the CurrentWeather component. 
I ended with the design like that. I know it should not be like that and this is not how I wanted. I couldn't do the exact design due to my poor CSS skill.

Thank you so much Lucy all along the way for teaching and guiding me!! I have learned a lot from you. I started final project on last Saturday therefore, I will focus on final project now. :)

Reply:
Good luck with it all @zkhing 😊